src/components/GlobalErrorBoundary/index.tsx

```diff
@@ -43,29 +43,35 @@ const LinkContent = styled.div`
   }
 `
 
-const GlobalErrorBoundaryFallback: FallbackRender = ({ error, componentStack }) => {
-  //  When loading app during release, chunk load failure may occur
+export const lastFallbackReloadKey = 'SAFE__lastFallbackReload'
```

Comment:
If it's just for the sake of tests, maybe it could be better to hard-code it in the test. What we export is the public API of a module, and we probably don't want other modules to mess with our stored value.